hil/ext/switches/common.py

```diff
@@ -53,3 +56,36 @@ def parse_vlans(raw_vlans):
             vlan_list.append(num_str)
 
     return vlan_list
+
+
+def modify_port_common(self, port, channel, new_network):
```

Comment:
Rather than making this a stand-alone function and assigning it in the class definitions, it would be more idiomatic for it to be a mixin class.

Reply:
oh right, that's the path I am on right now, didn't know it's called mixin class. Will push it in the next commit. I am factoring out more stuff.